### DISCUSSION TITLE
"Show answers" toggle like furigana?

### DISCUSSION THREAD

**Discussion by PaulWestphal** [SethClydesdale/genki-study-resources#132] [Ideas]
Sometimes I'm unsure and want to check right away. It's really cumbersome to check answers right now. Having a toggle to quickly check answers would be so much easier, no?

**Comment by SethClydesdale** [comment]
Hello @PaulWestphal, thank you for the suggestion!

It's possible, though may require a lot of work due to the multiple exercise types available. I will, however, note it down to look into in the future when I've more time to dedicate to feature implementations. At the current time though, the only exercise type that offers a similar feature would be the written exercises (workbook only for 2nd ed. OR workbook/practice for 3rd ed.), which allows you to preserve your answers when trying again after checking, but only if there's an incorrect answer.
